Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.7.3
+
+RUN gem install bundler
+
+# install node 14
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
```

Comment:
If you're going to install node using a helper script retrieved via curl/wget, it should a) be a fixed version and b) have its checksum validated (to help prevent external tampering)